tidb-cloud/release-notes-2022.md

```diff
@@ -8,6 +8,33 @@ aliases: ['/tidbcloud/beta/supported-tidb-versions','/tidbcloud/release-notes']
 
 This page lists the release notes of [TiDB Cloud](https://www.pingcap.com/tidb-cloud/) in 2022.
 
+## November 8, 2022
+
+* Support point-in-time recovery (PITR).
+
+    PITR supports restoring data of any point in time to a new cluster. It can be used to:
+
+    * Reduce RPO in disaster recovery.
+    * Resolve cases of data write errors by restoring point in time that is before the error event.
+    * Audit the historical data of the business.
+
+    To use the PITR feature, make sure that your TiDB cluster version is at least v6.3.0 and the TiKV node size is at least 8C/16G.
+
+    In addition, with PITR, TiDB Cloud supports setting the backup storage region for clusters located in Tokyo and Osaka regions, so that backup data can be stored in both Tokyo and Osaka, which facilitates data safety and faster recovery. When you restore data, you can choose either region for the new cluster.
```

Comment:
```suggestion
    In addition, with PITR, you can set the backup storage region for clusters located in Tokyo and Osaka regions, so that backup data can be stored in both Tokyo and Osaka, which facilitates data safety and faster recovery. When you restore data, you can choose either region for the new cluster.
```